plugins/modules/remote_management/oneview/oneview_enclosure_info.py

```diff
@@ -153,7 +155,11 @@
 
 
 class EnclosureInfoModule(OneViewModuleBase):
-    argument_spec = dict(name=dict(type='str'), options=dict(type='list'), params=dict(type='dict'))
+    argument_spec = dict(
+        name=dict(type='str'),
+        options=dict(type='list', elements='dict'),
```

Comment:
```suggestion
        options=dict(type='list', elements='raw'),
```
This can be both `str` and `dict` according to the examples. (Adding `raw` here does not need any changelog fragment, since it does no validation.)

Reply:
commit coming up